src/schema/schema.ts

```diff
@@ -175,6 +176,7 @@ if (!ENABLE_ECOMMERCE_STITCHING) {
   stitchedMutations.submitOrder = SubmitOrderMutation
 
   stitchedMutations.ecommerceCreateOrderWithArtwork = CreateOrderWithArtworkMutation
+  stitchedMutations.ecommerceCreateOfferOrderWithArtwork = CreateOfferOrderWithArtworkMutation
```

Comment:
I noticed that this was previously named `ecommerceCreateOfferOrderWithArtworkMutation`. Looks like we're naming the others without the `Mutation` suffix which seems fine (and also shortens it slightly).